Library/Formula/meanwhile.rb

```diff
@@ -0,0 +1,299 @@
+class Meanwhile < Formula
+  desc "Implementation of Lotus Sametime Community Client protocol"
+  homepage "http://meanwhile.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/meanwhile/meanwhile/1.0.2/meanwhile-1.0.2.tar.gz"
+  sha256 "38a8059eaa549cbcb45ca0a5b453e9608ceadab360db2ae14581fb22ddabdf23"
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+
+  patch :DATA
```

Comment:
The [Ubuntu repositories have meanwhile](https://launchpad.net/ubuntu/+source/meanwhile). From their [changelog](https://launchpad.net/ubuntu/+source/meanwhile/+changelog) at `1.0.2-4.1
`:

```
Fix "Including individual glib headers no longer supported":
    apply patch from Michael Biebl
```

The [Fedora repositories also have meanwhile](https://apps.fedoraproject.org/packages/meanwhile), but with a different minor version from the upstream source.